GCC will define __ARM_ARCH_4T__ when building with "-march=armv4t" so we can
check this to turn off the use of 'clz' instructions, which otherwise would
cause compile errors like "selected processor does not support ARM mode 
`clz r3,r0'". 

Upstream-Status: Accepted (should be in the next cogl stable release)

Signed-off-by: Wolfgang Denk <wd@denx.de>
Signed-off-by: Joshua Lock <josh@linux.intel.com>

Index: cogl-1.8.2/cogl/cogl-fixed.c
===================================================================
--- cogl-1.8.2.orig/cogl/cogl-fixed.c
+++ cogl-1.8.2/cogl/cogl-fixed.c
@@ -626,7 +626,7 @@ cogl_fixed_sqrt (CoglFixed x)
 	/*
 	 * Find the highest bit set
 	 */
-#if defined (__arm__)
+#if defined (__arm__) && !defined(__ARM_ARCH_4T__)
 	/* This actually requires at least arm v5, but gcc does not seem
 	 * to set the architecture defines correctly, and it is I think
 	 * very unlikely that anyone will want to use clutter on anything
